playbooks/playbook.yaml

```diff
@@ -61,6 +62,12 @@
         tasks_from: viya-monitoring
       tags:
         - viya-monitoring
+    - name: baseline role uninstall
+      include_role:
+        name: baseline
+      when: ('baseline' in ansible_run_tags) and ('uninstall' in ansible_run_tags)
+      tags:
+        - baseline
```

Comment:
I believe this line can be removed as the `when` clause is now taking care of this.